content/getstarted/02-item.md

```diff
@@ -1,56 +1,193 @@
 +++
-description = "Deploying Kubevirt with a CSI plugin can be tricky, there are multiple steps: setting Access Control of Kubelet, Kubernetes API, and the CSI plugin services, restarting of pods, and creating the manifests to properly deploy the controller and node CSI services.  This guide presents an easy way to try Kubevirt with Ember CSI."
+description = "Deploying KubeVirt with a CSI plugin can be tricky, there are multiple steps: setting Access Control of Kubelet, Kubernetes API, and the CSI plugin services, restarting of pods, and creating the manifests to properly deploy the controller and node CSI services.  This guide presents an easy way to try KubeVirt with Ember-CSI."
 thumbnail = "images/02-getstarted-thumb.jpg"
 image = "images/02-getstarted.jpg"
-title = "KubeVirt and Ember CSI"
-slug = "kubevirt"
-author = "Gorka Eguileor"
+title = "KubeVirt and Ember-CSI"
+slug = "ember-kubevirt"
+author = "Kiran Thyagaraja"
 draft = false
 hidesidebar = true
 publishDate=2018-08-02T19:05:52+02:00
+lastmod=2019-03-20T11:30:57-06:00
 weight = 2
 +++
-To use the Ember-CSI plugin on Kubevirt we need to make changes to our deployment.
+[KubeVirt](https://kubevirt.io) is a virtual machine management add-on for [Kubernetes](https://kubernetes.io). It allows users to run VMs alongside containers in the their Kubernetes or [OpenShift](https://www.openshift.com) clusters. This document describes a quick way to deploy either Kubernetes or OpenShift, KubeVirt and [Ember-CSI](https://ember-csi.io).
 
-The easiest way to make the changes is to copy this kubevirt directory into a directory of the kubevirt repository, and run the `csi_up.sh` script from there after the cluster is up and running.
+The [Ember-CSI-KubeVirt](https://github.com/embercsi/ember-csi-kubevirt) repository provides a seamless wrapper integrating the deployments of OpenShift or Kubernetes, KubeVirt, [Containerized Data Importer (CDI)](https://github.com/kubevirt/containerized-data-importer), Ember-CSI and a Ceph demo cluster. Ember-CSI is deployed using the [Ember-CSI-Oprator](https://github.com/embercsi/ember-csi-operator). The operator ensures that all of Ember-CSI's Kubernetes objects such as StatefulSets, DaemonSets, StorageClasses, RBAC rules etc are properly configured and running. The Ceph-demo cluster deploys an empty directory backed Ceph cluster within a namespace called `ceph-demo`. KubeVirt's repo's scripts provide an easy way to deploy OpenShift or Kubernetes using the `make cluster-up` scripts. In addition, CDI is deployed, which provides a declarative way to build Virtual Machine Disks on PVCs for Kubevirt VMs. Finally the Ember-CSI-KubeVirt repository ties all these deployments together into an easy `make all` command. 
```

Comment:
The paragraph is too long, probably best to split it.

Reply:
Yes. Agreed.